### ISSUE TITLE
How to synchronize the video and ppg signal

### ISSUE THREAD

**Issue by yuhongjiu8888** [partofthestars/LGI-PPGI-DB#6]
How do the video and ppg signal synchronize, they are different devices

**Comment by partofthestars** [comment]
resample it to a common sample rate

**Comment by GyutaeHwang** [comment]
Hello, thanks for sharing the data.
I wonder when the video and signal synchronization start point.
Need to synchronize the first frame of the video with the first recorded signal?
It's confusing because videos are longer than about 2.6 seconds of signal.

Thank you

**Comment by Nickkzz97** [comment]
Yes even after resampling the no of PPG data points are more than that of no of frames!

**Comment by partofthestars** [comment]
id1.zip alex gym:
xml log:
cms: 24116 -> 60 fps: 244116/60=401,93 sec
camera: 9946 -> 25 fps: 9946/25=397,84 sec
